deployment/arm/deploy-cosmosDB.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Similar comment to above regarding including this with the other global resources.

Reply:
I initially added this as part of the global resource but then figured we need to set cosmosdb account url and key to appsettings of App service. Based on the current deployment strategy, Regional resources including App service is deployed first and then global resources are deployed next. I am deploying cosmos first so that I can use the output values to set in App Service. I also would like to have all the global resources deployed in one stage including the cosmosdb. There are several ways to handle this, lets discuss it offline and then I`ll make all the necessary changes.